src/include/migraphx/compile_src.hpp

```diff
@@ -43,12 +43,11 @@ struct src_file
 
 struct src_compiler
 {
-    std::string compiler                      = "c++";
-    std::string flags                         = "";
-    std::string output                        = "";
-    std::string launcher                      = "";
-    std::string out_ext                       = ".o";
-    std::function<fs::path(fs::path)> process = nullptr;
+    std::string compiler = "c++";
```

Comment:
`process` should not be removed., its not hcc-specific. It could be used to invoke a linker.

Reply:
It wasn't used anywhere that's why i removed it.

Reply:
Reverted change.